metaswitch/common/mib.py

```diff
@@ -38,6 +38,9 @@ def get_all_stats(self, columns):
         `columns` should be a list of the properties of the statistic that we
         want to parse.
         """
+        # Add INDEX to the list of fields to retrieve and parse - we
+        # need this in order to check whether a node is a statistic or not.
+        columns.append("INDEX")

```

Comment:
What if "INDEX" is already in the list of columns?

I'm really not sure about the idea of this columns parameter - I'd normally just get all of the columns trading a small bit of perf for an improved API for the Statistic object.